package.json

```diff
@@ -69,6 +69,6 @@
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-vaadin": "^0.2.10",
-    "typescript": "4.3.5"
+    "typescript": "^4.4.3"
```

Comment:
Was it added intentionally? 🤔

Reply:
I had to lock the version in the previous PR because it failed there - see [the build](https://github.com/vaadin/web-components/runs/3673224391).
Now we upgrade to the latest TypeScript where the problem can be reproduced.

Reply:
Understood.